jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannelState.java

```diff
@@ -738,15 +739,15 @@ else if (failure instanceof UnavailableException)
         try(Locker.Lock lock= _locker.lock())
         {
             if (LOG.isDebugEnabled())
-                LOG.debug("onError {} {}",toStringLocked(),failure);
+                LOG.debug("onError {} {}",toStringLocked(),th);
             
             // Set error on request.
             if(_event!=null)
             {
-                _event.addThrowable(failure);
+                _event.addThrowable(th);
                 _event.getSuppliedRequest().setAttribute(ERROR_STATUS_CODE,code);
-                _event.getSuppliedRequest().setAttribute(ERROR_EXCEPTION,failure);
-                _event.getSuppliedRequest().setAttribute(RequestDispatcher.ERROR_EXCEPTION_TYPE,failure==null?null:failure.getClass());
+                _event.getSuppliedRequest().setAttribute(ERROR_EXCEPTION,th);
+                _event.getSuppliedRequest().setAttribute(RequestDispatcher.ERROR_EXCEPTION_TYPE,th==null?null:th.getClass());
```

Comment:
I hate static imports, but since the other constants are statically imported, this one should be too. I prefer no static imports.